contracts/helpers/RebalancerFlashloanSwap.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.19;
+
+import "./ARebalancerFlashloan.sol";
+import { IERC20Metadata } from "oz/interfaces/IERC20Metadata.sol";
+import { RouterSwapper } from "utils/src/RouterSwapper.sol";
+
+/// @title RebalancerFlashloanSwap
+/// @author Angle Labs, Inc.
+/// @dev Rebalancer contract for a Transmuter with as collaterals a liquid stablecoin and an yield bearing asset
+/// using this liquid stablecoin as an asset
+contract RebalancerFlashloanSwap is ARebalancerFlashloan, RouterSwapper {
+    using SafeCast for uint256;
+
+    uint32 public maxSlippage;
+
+    constructor(
+        IAccessControlManager _accessControlManager,
+        ITransmuter _transmuter,
+        IERC3156FlashLender _flashloan,
+        address _swapRouter,
+        address _tokenTransferAddress,
+        uint32 _maxSlippage
+    )
+        ARebalancerFlashloan(_accessControlManager, _transmuter, _flashloan)
+        RouterSwapper(_swapRouter, _tokenTransferAddress)
+    {
+        maxSlippage = _maxSlippage;
+    }
+
+    /**
+     * @notice Set the token transfer address
+     * @param newTokenTransferAddress address of the token transfer contract
+     */
```

Comment:
Why do we need to override if there is only one implementation?

Reply:
as we add a modifier